doc/mysqlnd_azure_log.md

```diff
@@ -0,0 +1,124 @@
+# Diagnostic Log Configuration
+
+Log down some runtime information from mysqlnd\_azure.
+
+## Configuration
+
+### Log destination supported (mysqlnd\_azure.logOutput)
```

Comment:
To keep the headings on this page consistent for all the configuration. Change this to

 ###mysqlnd\_azure.logOutput
Use this property to set the destination path for your logs. For all the file paths mentioned below, except Apache and php-fpm's system error log, it need to be configured with right user permission, e.g. www-data for Apache. For relative file path, when working in cli, the father path is the current working directoy, when working with Apache, the father path is /.   We provide multiple options to store your logs .